src/main/java/OGF.java

```diff
@@ -0,0 +1,132 @@
+import java.util.Scanner;
+public class OGF {
+    private static final int MAX_ITEMS = 100;
+    private static int numItem;
+    private static final Task[] tasks = new Task[MAX_ITEMS];
+
+    private static boolean isRunning = true;
+    private static void printBreakLine(){
+        System.out.println(("____________________________________________________________"));
+    }
+
+    private static void printTaskAdded(Task task, int taskIndex){
+        System.out.println("Alright, adding this task to the list: ");
+        System.out.println(task);
+        System.out.printf("You have %d tasks in the list.%n", taskIndex+1);
+        printBreakLine();
+    }
+
+    private static boolean handleInput (String input) throws OGFException{
```

Comment:
Very long method. Avoid long methods and take corrective action if a method exceeds 30 lines of code.